tests/manage/storageclass/conftest.py

```diff
@@ -0,0 +1,26 @@
+import logging
+from ocs_ci.framework import config
+from ocs_ci.ocs.constants import MANAGED_SERVICE_PLATFORMS
+
+log = logging.getLogger(__name__)
+
+
+def pytest_collection_modifyitems(items):
+    """
+    A pytest hook to filter out storageclass tests
+    when running on openshift dedicated platform
+
+    Args:
+        items: list of collected tests
+
+    """
+    # Need to update the condition when MCG will get supported
```

Comment:
This statement is not appropriate for storage class tests. Please modify the statement.